subsys/bluetooth/mesh/net.c

```diff
@@ -711,9 +710,9 @@ static void bt_mesh_net_relay(struct net_buf_simple *sbuf,
 		transmit = bt_mesh_net_transmit_get();
 	}
 
-	buf = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_ADV_TAG_RELAY,
+	adv = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_ADV_TAG_RELAY,
 				 transmit, K_NO_WAIT);
-	if (!buf) {
+	if (!adv) {
 		LOG_DBG("Out of relay buffers");
```

Comment:
Is this comment relevant after your changes?

Reply:
Done